runtime/interpreter/interpreter.go

```diff
@@ -2688,13 +2688,22 @@ var ConverterDeclarations = []ValueConverterDeclaration{
 		nestedVariables: []struct {
 			Name  string
 			Value Value
-		}{{
-			Name: sema.AddressTypeFromBytesFunctionName,
-			Value: NewUnmeteredHostFunctionValue(
-				sema.AddressConversionFunctionType,
-				AddressFromBytes,
-			),
-		}},
+		}{
+			{
+				Name: sema.AddressTypeFromBytesFunctionName,
+				Value: NewUnmeteredHostFunctionValue(
+					sema.AddressConversionFunctionType,
+					AddressFromBytes,
+				),
+			},
+			{
+				Name: sema.AddressTypeFromStringFunctionName,
+				Value: NewUnmeteredHostFunctionValue(
+					sema.AddressConversionFunctionType,
```

Comment:
Why does this have the same type as the function above?

Reply:
Done, thanks for catching! I made the same mistake in the `Address.fromBytes` implementation. Fixed both of them.